jupyter-docker/scripts/extension/jupyter_install_lab_extension.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This file should be removed I think, there is no `jupyter-docker` directory anymore